### PR TITLE
Dpl 1075 library edit volume check

### PR DESCRIPTION
**Pull request by seenanair**
This PR includes:

- Displaying a "used volume" label when the user edits a library.
- Preventing the user from entering a value for the library volume less than the used volume.

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

**Review by BenTopping — CHANGES_REQUESTED** [review]
Tested and looks really good. 

The forced minimum value is quite aggressive and it makes it quite tricky to change the volume as it doesn't let you delete values and retype easily. Can we debounce it or something or is that a tradeoff of using min?

**Comment by seenanair** [comment]
> Tested and looks really good.
> 
> The forced minimum value is quite aggressive and it makes it quite tricky to change the volume as it doesn't let you delete values and retype easily. Can we debounce it or something or is that a tradeoff of using min?

Yes, you are right. I felt it us very forceful (this is what I mentioned in https://github.com/sanger/traction-service/pull/1288 comment). I will look into it. Probably we can have a chat about it next week.

File: src/components/pacbio/PacbioLibraryForm.vue [hunk 1]
```diff
@@ -26,12 +26,34 @@
...
+            id="library-used-volume-div"
+            class="justify-end contents-end px-1 relative"
+            @mouseover="hover = true"
+            @mouseleave="hover = false"
+          >
+            <div
+              v-if="hover"
+              id="library-used-volume-tooltip"
+              class="text-sm px-1 bg-gray-700 text-gray-100 absolute rounded bg-opacity-50 shadow-xl left-0"
+              :style="{ top: '-25px' }"
```
[the remark this answers is not in the document]
Comment (seenanair) [reply src/components/pacbio/PacbioLibraryForm.vue:42]: I tried, but didn't work. I think it needs bit more work with tailwind config update. I will check it again. 👍

File: src/components/pacbio/PacbioLibraryForm.vue [hunk 2]
```diff
@@ -204,6 +233,20 @@ const provider = async () => {
   }
 }
 
+/**
+ * Watches for changes in the volume property of the formLibrary object.
+ * If the new volume is less than the used_volume, it resets the volume to the used_volume.*/
+watch(
```
[the remark this answers is not in the document]
Comment (seenanair) [reply src/components/pacbio/PacbioLibraryForm.vue:239]: Unfortunately :min doesn't prevent user from typing in values directly into textfield. It only sets the min for the spinner control.

**Review by BenTopping — DISMISSED** [review]
Looks good 👌

**Review by BenTopping — APPROVED** [review]

**Pull request merged**